src/Pages/Login/Login.js

```diff
@@ -1,9 +1,143 @@
 import React from "react";
 import { withRouter } from "react-router-dom";
+import "./Login.scss";
+import "../../Styles/common.scss";
 
 class Login extends React.Component {
+  state = {
+    email: "",
+    password: "",
+    token: "",
+    checked: "",
+  };
+
+  handlerlogin = () => {
+    fetch("http://10.58.4.0:8000/account/sign-in", {
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.password,
+        token: this.state.token,
```

Comment:
```suggestion
  handlerlogin = () => {
    const { email, password, token } = this.state;
    fetch("http://10.58.4.0:8000/account/sign-in", {
      method: "POST",
      body: JSON.stringify({
        email,
        password,
        token,
```
1. 구조 분해 할당을 사용하여 변수명을 줄여줄 수 있습니다.
2. 객체의 키와 값의 이름이 같다면 단축 속성명을 사용할 수 있습니다. (참고 : https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Object_initializer)

Reply:
수정완료